README.md

```diff
@@ -36,6 +36,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
   <tr>
     <td align="center"><a href="https://github.com/kfahn22"><img src="https://avatars.githubusercontent.com/u/65121394?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Kathy</b></sub></a><br /><a href="#content-kfahn22" title="Content">🖋</a></td>
     <td align="center"><a href="https://github.com/dipamsen"><img src="https://avatars.githubusercontent.com/u/59444569?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Dipam Sen</b></sub></a><br /><a href="#content-dipamsen" title="Content">🖋</a></td>
+    <td align="center"><a href="https://github.com/lobertrand"><img src="https://avatars.githubusercontent.com/u/37326143?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Loïc Bertrand</b></sub></a><br /><a href="https://github.com/CodingTrain/thecodingtrain.com/commits?author=lobertrand" title="Code">💻</a> <a href="https://github.com/CodingTrain/thecodingtrain.com/pulls?q=is%3Apr+reviewed-by%3Alobertrand" title="Reviewed Pull Requests">👀</a> <a href="#design-lobertrand" title="Design">🎨</a> <a href="#content-lobertrand" title="Content">🖋</a></td>
```

Comment:
```suggestion
    <td align="center"><a href="https://github.com/lobertrand"><img src="https://avatars.githubusercontent.com/u/37326143?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Loïc Bertrand</b></sub></a><br /><a href="https://github.com/CodingTrain/thecodingtrain.com/commits?author=lobertrand" title="Code">💻</a> <a href="#content-lobertrand" title="Content">🖋</a></td>
```

@shiffman It would be good if merged like this, thanks :)